commit e1882ee8c541020ec590bf096508ca5b6d2ab944 (HEAD, refs/heads/wk-master)
Author: Werner Koch <wk at gnupg.org>
Date:   Fri Sep 12 10:33:16 2014 +0200

    Fix a prototype
    
    * src/gpgrt-int.h: s/off_t/gpgrt_off_t/.

	Modified   src/gpgrt-int.h
diff --git a/src/gpgrt-int.h b/src/gpgrt-int.h
index 0e6f69c..f97166f 100644
--- src/gpgrt-int.h
+++ src/gpgrt-int.h
@@ -105,9 +105,9 @@
 
 int _gpgrt_fflush (gpgrt_stream_t stream);
 int _gpgrt_fseek (gpgrt_stream_t stream, long int offset, int whence);
-int _gpgrt_fseeko (gpgrt_stream_t stream, off_t offset, int whence);
+int _gpgrt_fseeko (gpgrt_stream_t stream, gpgrt_off_t offset, int whence);
 long int _gpgrt_ftell (gpgrt_stream_t stream);
-off_t _gpgrt_ftello (gpgrt_stream_t stream);
+gpgrt_off_t _gpgrt_ftello (gpgrt_stream_t stream);
 void _gpgrt_rewind (gpgrt_stream_t stream);
 
 int _gpgrt_fgetc (gpgrt_stream_t stream);
